frontend/common/kubeflow-common-lib/projects/kubeflow/tsconfig.lib.json

```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "outDir": "../../out-tsc/lib",
     "declarationMap": true,
-    "target": "es2015",
```

Comment:
why no more target version? maybe `es2022` like below config?

Reply:
This actually got updated to es2020 when i updated to angular 14. But then got deleted when i updated to v15. Not sure why 100%, Ill look into the cause for this because you are right that this might be suspicious. 

Oh actually now that I take a look at it, this file extends "../../tsconfig.json", which has the "target" property already(with value es2022)